src/components/Dialog/shared.tsx

```diff
@@ -29,10 +29,8 @@ export function Header({
         a.border_b,
         t.atoms.border_contrast_medium,
         t.atoms.bg,
-        web([
```

Comment:
what does this change? before/after?

Reply:
`a.overflow_hidden` was used to clip the corners of the header, but if we can't use that we need to use a border radius on the header itself instead